src/power_menu.rs

```diff
@@ -33,32 +33,53 @@ pub struct CliArgs {
     /// Show the menu items and exit
     #[arg(short, long)]
     pub list_items: bool,
+
+    /// Switch to elogind
+    #[arg(short, long, default_value = "false")]
+    pub elogind: bool,
+}
+
+// this should probably be somewhere else
+pub enum SessionManager {
+    Elogind,
+    Systemd,
+}
+
+impl std::fmt::Display for SessionManager {
```

Comment:
I'd prefer not to couple the Display trait logic to the "command to execute" resolution. Maybe it could be better to move this logic to a function for instance

Reply:
Looking back at my thought process, your thinking seems more reasonable. I'm thinking maybe an `impl Into<&str> for SessionManager { ... }` block. In my opinion, it doesn't cause any coupling, because converting the enum value to a `&str` is something you would likely need to do anyway at some point.